### DISCUSSION TITLE
envoy_mail_specifique

### DISCUSSION THREAD

**Discussion by fccl-vandoeuvre** [JeromeDevome/GRR#321] [Q&A]
Bonjour,

J’expérimente actuellement l’outil, je cherche à mettre en place un champ additionnel permettant de définir un courriel, pour y envoyer une notification relative lors d’une inscription.
J’utilise la version 4.2.2 qui a priori a justement corrigé une anomalie à ce sujet (voir ici : https://github.com/JeromeDevome/GRR/issues/295), donc en théorie ça devrait fonctionner. Mais ce n’est pas le cas.

Le test SMTP est ok, l’envoie automatique aussi.
J’ai pas trouvé d’information dans la doc à ce sujet, probablement qu’elle n’a pas été mise à jour pour intégrer ce système.

J’aimerais avoir quelques précisions pour être certain que ce ne soit pas moi qui comprends pas : 
- S’agit-il bien d’une possibilité d’ajouter manuellement une adresse de courriel lié à aucun compte existant dans GRR ?
- Dans le menu des champs, je trouve « (envoy_mail_specifique)*(1)(2) » en intitulé de colonne, et « (envois_mail_spec_exp) » en petit 2. J’imaginais qu’il fallait mettre dans le champ de la colonne ce qui est indiqué dans le petit 2. J’ai testé avec guillemets, sans, ça ne fonctionne pas. J’ai tenté avec d’autres contenus étranges, un champ vide… Je ne sais pas en fait ce qu’il faut faire ?

Merci bien pour la lecture.

**Comment by JeromeDevome** [comment]
Bonjour,
C'est juste une adresse mail qu'il faut saisir sans rien d'autre. L’envoi de mail s'active si le champ comporte une valeur et différent de 0.

**Reply by fccl-vandoeuvre** [reply]
Bonjour,

Merci pour la réponse rapide.

J’ai donc ajouté une adresse réelle dans le champ « (envoy_mail_specifique)*(1)(2) », puis ai écrit « 1 » lors d’une inscription sur un agenda de test. Le courriel est donc bien expédié à l’adresse indiquée.

Cependant, ce n’est pas cet usage qui m’intéresse, mais la possibilité d’indiquer à la volée lors d’une inscription une adresse de courriel, pour inscrire un usager et que celui-ci puisse recevoir un courriel de notification.

Si cela vous intéresse, je peux sonder les possibilités de développer courant 2024 une modif pour permettre ce dont j’ai besoin. À priori il ne manque que cela pour un usage concret dans ma collectivité.

Bien cordialement, Axel.
